src/main/webapp/profile.js

```diff
@@ -13,15 +13,21 @@
 // limitations under the License.
 
 function onProfileLoad() {
+  supplyLogoutLink();
   prepareFormValidation();
 }
 
-// Allows certain fields in the profile to be edited, hides edit button, and displays update button. 
-
-function onProfileLoad() {
-  supplyLogoutLink();
+// Submits profile form to Datastore.
+function submitProfileForm(methodType, redirectUrl) {
+  fetch(`/person?user-email=${$("#user-email").val()}&first-name=${$("#first-name-field").val()}`+
+    `&last-name=${$("#last-name-field").val()}&company=${$("#company-field").val()}&job=`+
+    `${$("#job-field").val()}&linkedin=${$("#linkedin-field").val()}`, 
+    {method: methodType})
+    .then(window.location.replace(redirectUrl));
```

Comment:
It probably make sense to explicitly handle any errors here and alert to the user if this fails.